Source/Core/Core/AchievementManager.h

```diff
@@ -64,8 +71,14 @@ class AchievementManager
     } remote_unlock_status = UnlockType::LOCKED;
     u32 session_unlock_count = 0;
     u32 points = 0;
+    BadgeStatus locked_badge;
+    BadgeStatus unlocked_badge;
   };
 
+  inline static const std::string GRAY = "transparent";
```

Comment:
```suggestion
  constexpr std::string_view GRAY = "transparent";
```

Ditto for the others. This'll get rid of some static constructors